config/locales/client.en.yml

```
@@ -499,6 +499,7 @@ en:
         no_value: "Resume editing"
       dropdown:
         title: "Open the latest drafts menu"
+        untitled: "Undefined title"
```

```suggestion
        untitled: "Untitled topic draft"
```

Maybe a better default for most users? `Undefined` sounds a little code-y.

---

I guess we would need to change this based on post drafts or topic drafts?

---

Hmm... under what conditions would a post draft have no title?

---

Could also shorten this to "Untitled draft". That'll be good for either topics/posts. And for communities that don't call topics topics (not many, but they exist!)

---

I agree on "Untitled draft". It's right next to the "New topic" button, so the "topic" part is pretty clear.